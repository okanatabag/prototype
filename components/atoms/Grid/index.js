import styled from "styled-components"

const Grid = styled.div`
    display: flex;
    align-items: center;
    flex-flow: wrap;
`;

export default Grid;