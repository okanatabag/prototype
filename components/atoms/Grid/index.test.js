<<<<<<< HEAD
import React from "react";
import renderer from "react-test-renderer";
import Grid from ".";

it("Grid renders correctly", () => {
	const tree = renderer
		.create(
			<Grid>
				<button type="button">test1</button>
				<button type="button">test1</button>
			</Grid>
		)
		.toJSON();
	expect(tree).toMatchSnapshot();
});
=======
import React from 'react';
import renderer from 'react-test-renderer';
import Grid from '.'
it('Grid renders correctly', () => {
    const tree = renderer
      .create(<Grid><button>test1</button><button>test1</button></Grid>)
      .toJSON();
    expect(tree).toMatchSnapshot();
});
>>>>>>> b9cde6f17d12c509c1e9bb7145254a0b4fa476c1
