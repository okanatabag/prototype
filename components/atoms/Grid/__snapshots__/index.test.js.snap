// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid renders correctly 1`] = `
<div
  className="Grid-sc-1ebmn77-0 jjKtRp"
>
  <button>
    test1
  </button>
  <button>
    test1
  </button>
</div>
`;
